.storybook/preview.tsx

```diff
@@ -0,0 +1,50 @@
+import { IntlProvider } from 'react-intl';
+
+import features from '../examples/src/features';
+
+import '../src/styles/variables';
+import '../src/styles/base.scss';
+import {reactIntl} from "./reactIntl";
+import {ProjectAnnotations, Renderer} from "@storybook/types";
+
+global.FEATURE_FLAGS = global.FEATURE_FLAGS || features; // cannot use the word FEATURES, storybook uses it
+global.FILE_ID = global.FILE_ID || '415542803939';
+global.FOLDER_ID = global.FOLDER_ID || '69083462919';
+global.TOKEN = global.TOKEN || 'P1n3ID8nYMxHRWvenDatQ9k6JKzWzYrz';
+
+global.IntlProvider = IntlProvider;
```

Comment:
sorry I was referring to the comment on `global.FEATURE_FLAGS`. I think the other ones are still valuable:
```suggestion
// Constants	
global.FEATURE_FLAGS = global.FEATURE_FLAGS || features;
global.FILE_ID = global.FILE_ID || '415542803939';
global.FOLDER_ID = global.FOLDER_ID || '69083462919';
// NOTE: The token used is a readonly token accessing public data in a demo enterprise. DO NOT PUT A WRITE TOKEN	
global.TOKEN = global.TOKEN || 'P1n3ID8nYMxHRWvenDatQ9k6JKzWzYrz';

// Components	
global.IntlProvider = IntlProvider;
```